wren-ai-service/eval/preparation.py

```diff
@@ -45,51 +48,55 @@ def gen_new_mdl_from_csv(mdl_path: str, csv_path: str):
     with open(mdl_path) as file:
         mdl = orjson.loads(file.read())
 
-    with open(csv_path) as file:
-        csv_data = [line.strip().split(",") for line in file.readlines()]
+    with open(csv_path, newline="\n") as file:
+        csv_data = csv.reader(file)
 
-    csv_data_by_table = {}
+        csv_data_by_table = {}
```

Comment:
nitpick: L54-L99 don't need to be inside with block

Reply:
Fixed, thanks